changelog/v1.6.27/docker-compose-file-docs.yaml

```
@@ -0,0 +1,4 @@
+changelog:
+  - type: FIX
```

```suggestion
  - type: NON_USER_FACING
```
docs changes should be non user facing so they don't render in the product changelog